server/channels/store/store.go

```diff
@@ -485,6 +486,7 @@ type UserStore interface {
 	IsEmpty(excludeBots bool) (bool, error)
 	GetUsersWithInvalidEmails(page int, perPage int, restrictedDomains string) ([]*model.User, error)
 	InsertUsers(users []*model.User) error
+	RefreshPostStatsForUsers() error
```

Comment:
It's a bit debatable if this should be in poststore or userstore. I am leaning towards postStore. 0/5 though

Reply:
0/5 as well. Can move it if you'd like me to :)

Reply:
Two 0/5s cancel each other out :) Let's leave it then.